src/i18n/locales/en/translation.json

```
@@ -110,13 +110,28 @@
           "connection_failed": "Connection failed. Are your LND credentials correct?"
         }
       },
-      "lndhub": {
-        "title": "LNDHub (Bluewallet)",
+      "lndhub_bluewallet": {
+        "title": "Bluewallet",
         "description": "Connect to your Bluewallet mobile wallet",
         "page": {
-          "title": "Connect to LNDHub (BlueWallet)",
+          "title": "Connect to BlueWallet",
           "description": "In BlueWallet, choose the wallet you want to connect, open it, click on \"...\", click on Export/Backup to display the QR code and scan it with your webcam."
         },
+        "uri": {
+          "label": "BlueWallet Export URI"
+        },
+        "errors": {
+          "invalid_uri": "Invalid BlueWallet URI",
+          "connection_failed": "Connection failed. Is your BlueWallet URI correct?"
+        }
+      },
+      "lndhub_go": {
+        "title": "LNDHub",
+        "description": "Connect to your LNDHub mobile wallet",
```

```suggestion
        "description": "Connect to your LNDHub account",
```

LNDHub is not a mobile wallet.